js/data/bucket.js

```diff
@@ -81,7 +81,7 @@ function Bucket(options) {
  * @private
  */
 Bucket.prototype.addFeatures = function() {
-    for (var i = 0; i < this.features.length; i++) {
+    for (var i = this.features.length - 1; i >= 0; i--) {
```

Comment:
The reverse iteration here makes the rendered order of features more similar to the previous rendered order, but that previous order was arbitrary. I think it should be safe to remove this and update the render tests.